sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/ReflectionUtils.java

```diff
@@ -55,35 +56,36 @@ public static ReflectiveInvoker getMethodInvoker(Class<?> targetClass, Method me
      * to alleviate this issue, if {@code targetClass} is null {@link Method#getDeclaringClass()} will be used to infer
      * the class.
      * <p>
-     * {@code scopeToCore} is only when used when MethodHandles are being used and Java 9+ modules are being used. This
+     * {@code scopeToAzure} is only when used when MethodHandles are being used and Java 9+ modules are being used. This
```

Comment:
We should also change the variable name.
```suggestion
     * {@code scopeToGenericCore} is only when used when MethodHandles are being used and Java 9+ modules are being used. This
```